app/assets/javascripts/views/project_view.js

```diff
@@ -21,6 +27,51 @@ module.exports = Backbone.View.extend({
     });
   },
 
+  prepareColumns: function() {
+    // reset columns & columns
+    this.columns = {};
+    this.$el.html(this.template({ inverse_flow: this.model.get('inverse_flow') }));
+    $('#column-toggles').html('');
```

Comment:
Do `$('#column-toggles')` is descendant of `this.el` ? Avoid use `$` directly for select elements, use `this.$()`